examples/Image Validation/Readme.md

```diff
@@ -0,0 +1,72 @@
+Dropzone for images validating dimensions and type of dropped images
+
+```
+class BasicValidatedImage extends React.Component {
+  constructor() {
+    super()
+    this.state = { files: [] }
+  }
+
+  onDrop(accepted, rejected, dropZoneRef) => {
+    let errors = [];
+  
+    if (rejected.length) {
+      errors.push('Image is over 10 MB and is therefore rejected.');
+    } else {
+      if (accepted[0].type !== 'image/jpeg') {
+        errors.push('This image is not a jpg and is therefore rejected.');
+      } else {
+        var acceptedImg = accepted[0];
+        var image  = new Image();
+        image.addEventListener('load', () => {
+          if (image.width !== 2500) {
+            errors.push('This image must be exactly 2500 pixels wide.');
+          } else if (image.height !== 3000) {
+            errors.push('This image must be exactly 3000 pixels wide.');
+          }
+  
+          // display errors or do success thing
+          if (errors.length > 0) {
+            alert(errors.join(', '));
+          } else {
+            alert('these images are valid!');
+            //this will depend on what your app does so this part will potentially cause problems
+            this.setState({
+              files: [acceptedImg]
+            });
+          }
+        });
+        //make the instantiated image variable the acceptedImg blob, for ex:
+        // blob:http://localhost:3000/19230c37-944c-4088-9096-ef1cd54f7a2d
+        image.src = acceptedImg.preview;
+      }
+    }
+  };
+
+
+  render() {
+    return (
+      <section>
+        <div className="dropzone">
+          <Dropzone 
+            onDrop={this.onDrop.bind(this)}
+            maxSize={10000000}
```

Comment:
I want to deprecate those props and would appreciate if you could update the example to check the size in the `onDrop` method. Feasible?